docs/contributing/index.md

```diff
@@ -43,7 +43,7 @@ If you are new to open source projects, we recommend reading GitHub's [How to Co
 Do not open issues for general support questions as we want to keep GitHub issues for confirmed bug reports. Instead, open a discussion in the Q&A category. For more details on our support mechanisms and links to troubleshooting guides, refer to our [Support guidelines](../support/index.md).
 
 !!! note
-    Issues created for questions or unconfirmed bugs will be moved to GitHub discussions by the maintainers.
+Issues created for questions or unconfirmed bugs will be moved to GitHub discussions by the maintainers.
```

Comment:
@LalithVipulananthan You should add an empty line here.
![image](https://user-images.githubusercontent.com/31987104/165219998-50851ad9-54cf-4787-9f6d-bca73e173549.png)

Reply:
Applied in c809daeef376dc94c28bd96437d2415a1a74db5e